Xamarin.Forms.Platform.Android/Renderers/FormsTextView.cs

```diff
@@ -1,12 +1,13 @@
 using System;
 using Android.Content;
 using Android.Runtime;
+using Android.Support.V7.Widget;
 using Android.Util;
 using Android.Widget;
 
 namespace Xamarin.Forms.Platform.Android
 {
-	public class FormsTextView : TextView
+	public class FormsTextView : AppCompatTextView
```

Comment:
Breaking change. Is this necessary?

Reply:
Yes!
Like I commented before, to solve api19 bug we need to use appcompattextview as base

>Looking for #7205 I've checked that we need to use Android.Support.V7.Widget.AppCompatTextView
>So, any problem to change Xamarin.Forms.Platform.Android.FormsTextView base class?
>I've checked that only LabelRenderer inherits this class:

Reply:
A lot of third-party controls or custom renderers may rely on this being a TextView. This may also completely break any pre-AppCompat apps.

Any chance at all we can do this in a backwards compatible way instead?